lib/clAdapters/activeSync/base/Adapter.ts

```diff
@@ -0,0 +1,36 @@
+import { Adapter } from '../../base/index';
+import { AdapterCredentials } from '../../base/Adapter';
+
+/**
+ * `credentials` format:
+ * ```
+ * {
+ *   username: 'email@domain.com',
+ *   email: 'email@domain.com', // Typically same as username
+ *   password: 'PASSWORD',
+ * }
```

Comment:
Nit/non-blocking: Add `connectUrl` to this docblock so it shows in autocomplete.